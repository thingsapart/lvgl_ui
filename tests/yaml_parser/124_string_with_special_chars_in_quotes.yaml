# Tests that special YAML characters are ignored inside quoted strings.
text: "A string with [ and { and : and # inside it"
another: 'Another string with [ and { and : and # inside it'
