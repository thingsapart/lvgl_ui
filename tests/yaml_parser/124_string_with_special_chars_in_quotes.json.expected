[{
		"text":	"A string with [ and { and : and # inside it",
		"another":	"Another string with [ and { and : and # inside it"
	}]
